tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py34, py35, lint
+
+[testenv]
+setenv =
+; both temper-python and XBee modules have utf8 in their README files
+; which get read in from setup.py. If we don't force our locale to a
+; utf8 one, tox's env is reset. And the install of these 2 packages
+; fail.
+    LANG=en_US.UTF-8
+    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
+commands =
+     py.test -v --timeout=30 --cov --cov-report= {posargs}
+deps =
+     -r{toxinidir}/requirements_all.txt
+     -r{toxinidir}/requirements_test.txt
+
+[testenv:lint]
+basepython = python3
+commands =
+     flake8
+     pylint homeassistant
+     python script/gen_requirements_all.py validate
```

Comment:
I was pretty sure this is working as designed. We're trying to figure out requirements_all is out of sync, right? So having requirements_all.txt installed and missing something still makes it explode here (realistically the pylint also errors under those conditions).

If you have a scenario in mind this doesn't catch let me know.